<launch>
	<include file="$(find openbot_gazebo)/launch/gazebo.launch"/>
	<include file="$(find openbot_control)/launch/controller.launch"/> 
	<!--Launch the inverse kinematic -->
	<node pkg="modelos_cinematicos"
	 type="Demo_cinematica_Inversa_2G.py"
	 name="node_inverse_kinematic_2g"
	 output="screen"
	 >
	</node>
	<!--RQT_reconfigure for the dynamic configure-->
	<node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure"/> 
	<!--TF frame to know the Position of the effector-->
	<node pkg="tf2_ros" type="static_transform_publisher" name="effector_link_tf" args="0 0 0.15 0 0 0 1 clamp_arm_link_1 effector_link_tf" />

</launch>
